test/evergreen.yml

```diff
@@ -4293,9 +4293,9 @@ buildvariants:
     - name: many-collection-test
       distros: ubuntu2004-wt-large
 
-- name: cppsuite-stress-tests
-  display_name: "Cppsuite Stress Tests"
-  batchtime: 480 # 3 times a day
+- name: cppsuite-stress-tests-ubuntu
+  display_name: "Cppsuite Stress Tests Ubuntu 20.04"
+  batchtime: 720 # twice a day
```

Comment:
With the two variants we'll now run 4x a day instead of 3x a day by reducing both to 2x a day.